wgpu-types/src/features.rs

```diff
@@ -1231,6 +1231,15 @@ bitflags_array! {
         ///
         /// [`Device::create_shader_module_passthrough`]: https://docs.rs/wgpu/latest/wgpu/struct.Device.html#method.create_shader_module_passthrough
         const EXPERIMENTAL_PASSTHROUGH_SHADERS = 1 << 52;
+
+        /// Enables using multiview where not all texture array layers are rendered to in a single render pass/render pipeline.
+        ///
+        /// Supported platforms
+        /// - Vulkan
+        /// - DX12
```

Comment:
DX12 isn't supported yet with this PR

For future reference: when actually adding DX12 support we should also add docs pointing support out to `MULTIVIEW` feature itself